ivy/functional/frontends/numpy/manipulation_routines/changing_kind_of_array.py

```diff
@@ -1,7 +1,2 @@
 # local
-# import ivy
-
-
-# TODO: define this as an alias of asarray in from_existing_data.py
-# def asarray(a, dtype=None, order=None, *, like=None):
-#     return ivy.asarray(a, dtype=dtype)
+from ivy.functional.frontends.numpy.creation_routines import asarray
```

Comment:
actually this might lead to a problem with flake8 linting. Is it saying unused import error? Maybe writing:

```
asarray = asarray
```

in the file will fix it, could you give this a try?

Reply:
yeah it's flagging it. Didn't catch it in my commit cause I merged from master (to fix the merge conflict) and pulled the change to my branch. Do we have to state the variable assignment explicitly in the docs as well?

Reply:
Sorry I'm not so clear, on the comment about handling it here  `ivy/functional/frontends/numpy/manipulation_routines/__init__.py`. Will we still import it and assign to a variable?

Reply:
no we can just add this into the `__init__.py` file and it should work I believe:
```
from ivy.functional.frontends.numpy.creation_routines import asarray
```